tests/test_2024_11_26.py

```diff
@@ -0,0 +1,379 @@
+    assert interface.AllowedRecipientRegistry(atc_allowed_recipients_registry).isUnderSpendableBalance(atc_spendable_balance_after, 3)
+    assert atc_spendable_balance_after == atc_spend_limit_after_expected
+    limit_test(easy_track,
+               int(atc_spendable_balance_after / (10**18)) * 10**6,
+               atc_trusted_caller_acc,
+               atc_top_up_evm_script_factory,
+               atc_multisig_acc,
+               stranger,
+               interface.Usdc(USDC_TOKEN),
+               2_000_000 * 10**6
+    )
+
+    # Item 2
+    pml_budget_limit_after, pml_period_duration_months_after = interface.AllowedRecipientRegistry(pml_allowed_recipients_registry).getLimitParameters()
+    pml_spend_limit_after = interface.AllowedRecipientRegistry(pml_allowed_recipients_registry).getPeriodState()[1]
+    pmlSpendableBalanceAfter = interface.AllowedRecipientRegistry(pml_allowed_recipients_registry).spendableBalance()
+    assert pml_budget_limit_after == pml_budget_limit_after_expected
+    assert pml_period_duration_months_after == 3
+    assert pml_spend_limit_after == pml_spend_limit_after_expected
+    assert interface.AllowedRecipientRegistry(pml_allowed_recipients_registry).isUnderSpendableBalance(pmlSpendableBalanceAfter, 3)
+    assert pmlSpendableBalanceAfter == pml_spend_limit_after_expected
+    limit_test(easy_track,
+               int(pmlSpendableBalanceAfter / (10**18)) * 10**6,
+               pml_trusted_caller_acc,
+               pml_top_up_evm_script_factory,
+               pml_multisig_acc,
+               stranger,
+               interface.Usdc(USDC_TOKEN),
+               2_000_000 * 10**6
+    )
+
+    # Item 3
+    stonks_steth_budget_limit_after, stonks_steth_period_duration_months_after = interface.AllowedRecipientRegistry(stonks_steth_allowed_recipients_registry).getLimitParameters()
+    assert stonks_steth_budget_limit_after == stonks_steth_budget_after_expected
+    assert stonks_steth_period_duration_months_after == 6
+
+    # Item 4
+    already_spent_amount_after, _, stonks_steth_period_start_after, stonks_steth_period_end_after = stonks_steth_allowed_recipients_registry.getPeriodState()
+    assert stonks_steth_period_start_after == stonks_steth_period_start_exptected
+    assert stonks_steth_period_end_after == stonks_steth_period_end_exptected
+    assert already_spent_amount_after == stonks_steth_new_spent_amount_expected
+    stonks_steth_spendable_balance_after = interface.AllowedRecipientRegistry(stonks_steth_allowed_recipients_registry).spendableBalance()
+    assert stonks_steth_spendable_balance_after == stonks_steth_budget_after_expected
+    limit_test(easy_track,
+               stonks_steth_spendable_balance_after,
+               stonks_steth_trusted_caller,
+               stonks_steth_top_up_evm_script_factory,
+               stonks_steth_contract,
+               stranger,
+               stETH_token,
+               5_000 * 10 ** 18
+    )
+
+    # Item 5
```

Comment:
I'd checked that all parameters are the same as before the vote and only the reward address changed.
It's clearer in my pov